notecard.api.json

```diff
@@ -6,71 +6,209 @@
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/file.stats.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/hub.get.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/hub.log.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/hub.set.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/hub.signal.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/hub.status.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/hub.sync.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/hub.sync.status.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/note.add.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/note.changes.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/note.delete.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/note.get.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/note.template.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/note.update.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/ntn.gps.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/ntn.reset.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/ntn.status.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/var.delete.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/var.get.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/var.set.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/web.delete.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/web.get.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/web.post.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/web.put.req.notecard.api.json"},
-        {"$ref": "https://raw.githubusercontent.com/blues/notecard-schema/master/web.req.notecard.api.json"}    ]
-}
+        {
```

Comment:
I prefer the old whitespace. Did an auto-formatter do this?

Reply:
Yeah